### PR TITLE
Feature/refactor feature files enrique

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: trello/api/features/negative/BoardNegative.feature [hunk 1]
```diff
@@ -2,8 +2,19 @@
...
   As a regular user, It manages the board, and user Gets a board.
 
-  Scenario: Gets a board by invalid Id
-    When  Sets a "GET" request to "/boards/idBoardNotValid"
-    And Sends request
+#  Scenario: Gets a board by invalid Id
+#    When  Sets a "GET" request to "/boards/idBoardNotValid"
+#    And Sends request
+#    Then Should return status code 400
+#    And Validates response message with message "invalid id"
```
Comment (Amet015) [inline trello/api/features/negative/BoardNegative.feature]: comments? please unComent

File: trello/api/features/negative/BoardNegative.feature [hunk 2]
```diff
@@ -2,8 +2,19 @@
...
+    Given Sets a "POST" request to "/boards/"
+      | key  | value |
+      | name |       |
+    When Sends request
     Then Should return status code 400
-    And Validates response message with message "invalid id"
+    And Validates response message with message "invalid value for name"
+    And Sets a "GET" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 400
```
Comment (Amet015) [inline trello/api/features/negative/BoardNegative.feature]: please add a new line here

File: trello/api/features/smoke/BoardSmoke.feature [hunk 3]
```diff
@@ -32,6 +32,8 @@ Feature: Board's Test
     And Sends request
     Then Should return status code 200
     And Validates response body with
+      | key    | value |
+      | _value | None  |
```
Comment (Amet015) [inline trello/api/features/smoke/BoardSmoke.feature:36]: why are you using _value?

Comment (Enrique-C) [reply trello/api/features/smoke/BoardSmoke.feature:36]: Json response
{
    "_value": null
}

File: trello/api/features/negative/BoardNegative.feature [hunk 4]
```diff
@@ -7,3 +7,14 @@ Feature: Negative test of Board
...
+  Scenario: Creates a new board with a name empty
+    Given Sets a "POST" request to "/boards/"
+      | key  | value |
+      | name |       |
+    When Sends request
+    Then Should return status code 400
+    And Validates response message with message "invalid value for name"
+    And Sets a "GET" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 400
```
Comment (Amet015) [inline trello/api/features/negative/BoardNegative.feature]: add new line here please

**Review by Amet015 — APPROVED** [review]

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**